docker-compose.yml

```
@@ -0,0 +1,39 @@
+  frontend:
+    container_name: frontend
+    build: ./frontend
+    env_file:
+      - ./frontend/.env.docker
+    ports:
+      - "5173:5173"
+
+  redis:
+    container_name: redis
+    restart: unless-stopped
+    image: redis:7.0.5-alpine 
```

yamllint

[error] trailing spaces

(trailing-spaces)